scripts/api_data_proxy/static_data.json

```diff
@@ -0,0 +1,59 @@
+[
```

Comment:
This is a neat idea. We should figure out how big (how many records) we want this to have. It should be a big enough sample of data.